schemas/context/media-timed-asset-reference.schema.json

```diff
@@ -21,11 +21,39 @@
           "format": "uri-reference",
           "description": "Identifier of the content, which can be used to tie back to other industry or CMS IDs."
         },
+        "dc:creator": {
+          "title": "Creator",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An entity primarily responsible for making the resource."
+        },
+        "dc:publisher": {
```

Comment:
Shouldn't these be references to the master DC grammar and not copied here?

Reply:
Changed this with a reference to DC grammar.